openedx/core/djangoapps/user_authn/views/tests/test_login.py

```diff
@@ -103,106 +105,106 @@ def test_login_fail_no_user_exists(self):
         nonexistent_email = u'not_a_user@edx.org'
         response, mock_audit_log = self._login_response(
             nonexistent_email,
-            'test_password',
-        )
-        self._assert_response(response, success=False,
-                              value='Email or password is incorrect')
-        self._assert_audit_log(mock_audit_log, 'warning', [u'Login failed', u'Unknown user email', nonexistent_email])
-
-    def test_login_fail_incorrect_email(self):
```

Comment:
Note for reviewers.

Test `test_login_fail_incorrect_email` removed as a dup of  `test_login_fail_no_user_exists`.